src/test/java/com/github/mauricioaniche/ck/MethodInvocationsTest.java

```diff
@@ -0,0 +1,208 @@
+package com.github.mauricioaniche.ck;
+
+import com.google.common.collect.Sets;
+import junit.framework.Assert;
+import org.apache.commons.lang.ArrayUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class MethodInvocationsTest extends BaseTest {
+    private static Map<String, CKClassResult> report;
+
+    @BeforeAll
+    public static void setUp() {
+        report = run(fixturesDir() + "/MethodInvocation");
+    }
+
+    //Test method invocations
+    private void testMethodInvocation(Map<String, Set<String>> methodInvocations, Set<CKMethodResult> methods){
```

Comment:
rename it to `assertMethodInvocation`, so it's clear that this is an assert method.

Reply:
Done